vars/MDSDToolsPipeline.groovy

```diff
@@ -0,0 +1,30 @@
+def call(body) {
+    AbstractMDSDToolsDSLPipeline {
+        agent_label = 'docker'
+
+        buildWithMaven {
+            version = '3.6.0'
+            jdkVersion = 11
+        }
+
+        constraintBuild {
+            timeLimitMinutes = 30
+            ramLimit = '4G'
+            hddLimit = '20G'
+        }
+        
+        skipDeploy ("${this.BRANCH_NAME}" != 'master')
+        skipNotification ("${this.BRANCH_NAME}" != 'master')
+        
+        deployUpdatesiteSshName 'web'
```

Comment:
buildWith and constraintBuild only support one setting, calling it mutliple times results in overriding the previous setting. The deployment stuff here does not define a deployment step but solely provides some default settings in case the Jenkinsfile specifies a deployUpdatesite step.